src/lib/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, Spacer } from "@artsy/palette"
+import { CommercialButtons_artwork } from "__generated__/CommercialButtons_artwork.graphql"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+import { BidButtonFragmentContainer as BidButton } from "./BidButton"
+import { BuyNowButtonFragmentContainer as BuyNowButton } from "./BuyNowButton"
+import { MakeOfferButtonFragmentContainer as MakeOfferButton } from "./MakeOfferButton"
+
+export interface CommercialButtonProps {
+  artwork: CommercialButtons_artwork
+  relay: RelayProp
+  editionSetID?: string
```

Comment:
Can we add a comment here describing why we're using a prop for edition set ID, and not including it in the fragment? It makes sense to me now, but won't in six months.